src/components/TipCard/TipCard.css

```diff
@@ -1,15 +1,44 @@
 .tip-card {
-  align-items: center;
   border: 2px solid black;
   border-radius: 10px;
   display: flex;
   flex-direction: column;
-  height: 30em;
   justify-content: space-between;
   padding: 5%;
-  width: 20em;
+  text-align: center;
+  height: 83%;
+  width: 85%;
+  margin-bottom: 20%;
 }
 
 .description {
   text-align: center;
 }
+
+.delete {
+  background-color: lightgoldenrodyellow;
+  border-radius: 30px;
+  padding: .5rem;
+  margin-bottom: 3%;
+}
+
+.delete:hover {
+  background-color: #056475;
+  color: white;
+  cursor: pointer;
+}
+
+.details {
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+}
+
+.details :nth-child(1) {
```

Comment:
Were we having an issue with just the right margin for the module and rating inside the TipCard? Like to see people employing the use of nth child in CSS. :)